content/md/productivity/get-started-with-the-rules-engine.md

```diff
@@ -575,10 +575,25 @@ If you want to select the activated products, set the value to "true". If you wa
 ## Completeness
 
 The possible operators for the `completeness` field are:  
-- =
-- !=
-- “>”
-- <
+- 'NOT EQUALS ON AT LEAST ONE LOCALE'
+- 'EQUALS ON AT LEAST ONE LOCALE'
+- 'GREATER THAN ON AT LEAST ONE LOCALE'
+- 'GREATER OR EQUALS THAN ON AT LEAST ONE LOCALE'
+- 'LOWER OR EQUALS THAN ON AT LEAST ONE LOCALE'
+- 'LOWER THAN ON AT LEAST ONE LOCALE'
+
+:::warning
+You probably used different operators (i.e: '!=', '=', '>', '>=', '<=', '<'). These operators are now deprecated as they were not clear enough. If you used them, please take a look at the following matching table to know which operator you should use now.
+:::
+
+| Deprecated operators | Operators to use  |
+|---|---|
+| !=  | 'NOT EQUALS ON AT LEAST ONE LOCALE'  |
+| =  | 'EQUALS ON AT LEAST ONE LOCALE'  |
+| >  | 'GREATER THAN ON AT LEAST ONE LOCALE' |
+| >=  | 'GREATER OR EQUALS THAN ON AT LEAST ONE LOCALE'  |
+| <=  | 'LOWER OR EQUALS THAN ON AT LEAST ONE LOCALE'  |
+| <  | 'LOWER THAN ON AT LEAST ONE LOCALE'  |
```

Comment:
there are matching operators for all locales as well: `GREATER_THAN_ON_ALL_LOCALES`, `GREATER_OR_EQUALS_THAN_ON_ALL_LOCALES`, `LOWER_OR_EQUALS_THAN_ON_ALL_LOCALES`, `LOWER_THAN_ON_ALL_LOCALES`